src/AWS.Deploy.CLI/Extensions/CustomServiceCollectionExtension.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using AWS.Deploy.CLI.Commands;
+using AWS.Deploy.CLI.Commands.TypeHints;
+using AWS.Deploy.CLI.Utilities;
+using AWS.Deploy.Common;
+using AWS.Deploy.Common.Extensions;
+using AWS.Deploy.Common.IO;
+using AWS.Deploy.Orchestration;
```

Comment:
fyi:  In the future for either testing or visual studio support, we'll likely want to move the Orchestrator and Common binding to a separate extension method.  So we can do this:

```csharp
public void ExampleBinding()
{
    // binding for cli
    new ServiceCollection()
        .AddOrchestratorBindings()
        .AddDeployToolCli();

    // binding for visual studio
    new ServiceCollection()
        .AddOrchestratorBindings()
        .AddDeployToolVisualStudio();

    // binding for integration test with mocked io
    new ServiceCollection()
        .AddOrchestratorBindings()
        .AddDeployToolIntegrationTest();
}

```